From f8f80882bf69caa62b556d76f27ad10418630471 Mon Sep 17 00:00:00 2001
From: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Date: Fri, 29 Sep 2023 14:35:56 +0200
Subject: [PATCH 0565/1141] ARM: configs: enable USB_HIDDEV in multiv7 addons
 fragment

Enable devices with a separate event interface on /dev/usb/hiddevX

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@foss.st.com>
Change-Id: I582c85071a7e11f709e28e9ec11e7e5f63a78858
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/330609
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/fragment-02-multiv7_addons.config b/arch/arm/configs/fragment-02-multiv7_addons.config
index c3650a026396..338cf511cc40 100644
--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -85,6 +85,7 @@ CONFIG_TOUCHSCREEN_EDT_FT5X06=y
 CONFIG_TOUCHSCREEN_GOODIX=y
 CONFIG_TYPEC_UCSI=m
 CONFIG_UCSI_STM32G0=m
+CONFIG_USB_HIDDEV=y
 CONFIG_USB_ONBOARD_HUB=y
 CONFIG_USELIB=y
 CONFIG_VFP=y
-- 
2.39.2

